### PR TITLE
Add code comments for posterity

### PR DESCRIPTION
**Pull request by nvie**
This PR leaves a few comments in places that were not obvious to me, but which Guillaume clarified for me on our pairing session earlier this week. Documenting these insights for posterity here.

### CODE REVIEW

File: packages/liveblocks-client/src/room.ts [hunk 1]
```diff
@@ -178,7 +178,11 @@ export type Effects = {
 };
 
 type Context = {
+  /**
+   * @deprecated Use `roomId` instead.
+   */
   room: string;
+  roomId: string;
```
Comment (nvie) [inline packages/liveblocks-client/src/room.ts:185]: I read `context.room` somewhere, and assumed it was of type `room: Room`, but actually it was a `room: string` when I looked more closely, so changing the name. I'm duplicating the field here for now and deprecating it, but I'm not sure if that's even necessary. Seems to be a private field used only by the client itself?

Comment (nvie) [inline packages/liveblocks-client/src/room.ts:185]: I read `context.room` somewhere, and assumed it was of type `room: Room`, but actually it was a `room: string` when I looked more closely, so changing the name. I'm duplicating the field here for now and deprecating it, but I'm not sure if that's even necessary. Seems to be a private field used only by the client itself?

Comment (GuillaumeSalles) [reply packages/liveblocks-client/src/room.ts:185]: I didn't verify but I'm 99% sure that it's internal so it could be renamed.

Comment (nvie) [reply packages/liveblocks-client/src/room.ts:181]: Fixed in ea71ceaf7b9ecf24da748996b8cb243688904f61 👍

**Pull request merged**